WordPress/Classes/ViewRelated/Me/My Domains/MyDomainsViewController.swift

```diff
@@ -0,0 +1,191 @@
+import UIKit
+import Combine
+
+final class MyDomainsViewController: UIViewController {
+    enum Action: String {
```

Comment:
This `enum` is from this [PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/21701). Not sure why it shows as new code.